content/projects/data-science-specific/statistical-thinking/_index.md

```diff
@@ -44,36 +44,34 @@ Complete the DataCamp courses [Statistical Thinking in Python Part 1](https://ww
-5. Create a histogram of the CO2 emissions for 1990. Is this what you expected from your answers in questions 3 and 4?
-6. Calculate the standard deviation and standard error of the mean for CO2 emissions in 1990. How is the standard error different from the standard deviation?
-7. Create a line graph to show CO2 emissions in Brazil, Russia, China, India, the USA and South Africa over time. What does the graph tell you about the difference and change in C02 emissions in these countries?
-8. What is the mean and standard deviation for land area covered by forest in 1990? Why do you think the standard deviation is so large?
-9. Create histograms for land area covered by forest and percentage of area protected in 1990. Describe the distributions.
-10. Create a scatterplot with a regression line using `seaborn.regplot` to show the relationship between the proportion of land area covered by forest and the percentage of area protected in 2000.
+1. Load the [MDG data](MDG_Export_20191227.csv) into a pandas DataFrame. You will need the packages `matplotlib`, `numpy`, `seaborn`, `pandas` and `scipy`.
+2. How many different countries are represented? Save this number in a variable `number_of_countries`. 
+3. How many missing values do we have by country, year and series? You need to create 3 DataFrames - `missing_values_by_country_df` with column names `country` and `missing_values_count`, `missing_values_by_year_df` with column names `year` and `missing_values_count` and `missing_values_by_series_df` with column names `series` and `missing_values_count`
+4. Who are the top and bottom 5 countries in terms of CO2 emissions in 1990 and what are their emissions? Create DataFrames called `top_countries_co2_emmissions_df` and  `bottom_countries_co2_emmissions_df` with columns `Country` and `co2_emissions` and order the data from highest to lowest for `top_countries_co2_emmissions_df` and from lowest to highest for `bottom_countries_co2_emmissions_df`. How have these emissions changed by 2011?
+5. Check and correct any problems with the data. At this stage ensure that your dataset is in a DataFrame named: `mdg_df`
+6. Calculate the mean and median CO2 emissions for 1990. Save each answer in the variables `mean_co2_emmisions_1990` and `median_co2_emmisions_1990` respectively. Why do you think these values differ? 
+7. Calculate the minimum, maximum and interquartile range of the CO2 emissions for 1990. Please store these as variables named: `minimum_co2_emmisions_1990`, `maximum_co2_emmisions_1990` and `iqr_co2_emissions_1990` respectively. Using this information, as well as the mean and median calculated previously for this year, explain what this tells us about the distribution of CO2 emissions?
+8. Create a histogram of the CO2 emissions for 1990. Is this what you expected from your answers in questions 3 and 4? 
```

Comment:
rather:
"Is this the distribution you expected based on your answers from question x and x"

Also ensure the question numbers it refers to are correct